service/random/random.go

```diff
@@ -0,0 +1,76 @@
+package random
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+var (
+	defaultLenght = 30
+	valueMaxLife  = time.Hour
+	cleanupCycle  = time.Minute
+	characters    = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+)
+
+type value struct {
+	username  string
+	gameID    uuid.UUID
+	createdAt time.Time
+}
+
+type RandomGen struct {
+	r *rand.Rand
+	l int
+	s map[string]value
+}
+
+// New returns a new RandomGen
+func New() RandomGen {
+	r := RandomGen{
+		r: rand.New(rand.NewSource(time.Now().UnixNano())),
+		l: defaultLenght,
+		s: make(map[string]value),
+	}
+	go r.cleanup()
+	return r
+}
```

Comment:
RandomGen should listen to some context, to end cleanup when the server is shutting down for efficient resource cleanup.